app/build.gradle

```diff
@@ -5,25 +5,52 @@ android {
     buildToolsVersion "29.0.2"
     defaultConfig {
         applicationId "com.example.candor365"
-        minSdkVersion 28
-        targetSdkVersion 29
+        minSdkVersion 21
+        targetSdkVersion 28
```

Comment:
this might break things

Reply:
targetSdkVersion and BuildTools version should probably match. We could run into weird issues